.github/workflows/update-make-docs.yml

```
@@ -0,0 +1,15 @@
+name: Update `make docs` procedure
+on:
+  schedule:
+    - cron: '0 7 * * 1-5'
+  workflow_dispatch:
+jobs:
+  main:
+    if: github.repository == 'grafana/opentelemetry-docs'
```

Should this be `grafana/mimir`?

---

Good catch, thank you.

I was cleaning up a bunch of old PRs like this and forgot to update the copy-paste in this one.

```suggestion
    if: github.repository == 'grafana/mimir'
```